dojo/tools/cyclonedx/parser.py

```diff
@@ -167,7 +167,7 @@ def manage_vulnerability(
         cwes = self.get_cwes(vulnerability, ns)
         if len(cwes) > 1:
             # FIXME support more than one CWE
-            LOGGER.warning(
+            LOGGER.debug(
```

Comment:
Why do you suppress this message? Might be interesting for a production environment to see why there is no CWE in a finding. The same for the other parsers in this PR.